src/styles/index.scss

```diff
@@ -1,3 +1,112 @@
+@import "vars";
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: $text-second-color;
+}
+
+.card {
```

Comment:
Create separate scss files for each BEM block
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/f322ba63-fd5c-45b3-825b-d27b29807b6b)